msteams-platform/messaging-extensions/what-are-messaging-extensions.md

```diff
@@ -40,6 +40,20 @@ A message extension consists of a web service that you host and an app manifest,
 
 In the app manifest for Teams app, a single message extension is defined with up to 10 different commands. Each command defines a type, such as action or search and the locations in the client from where it's invoked. The invoke locations are compose message area, command bar, and message. On invoke, the web service receives an HTTPS message with a JSON payload including all the relevant information. Respond with a JSON payload, allowing the Teams client to know the next interaction to enable.
 
+## Message extension type
+
+If you don't already have a message extension, there are two ways to build one:
+
+* **API based message extension**: Using the Teams Toolkit, developers can easily create a API-backed ME from an existing API or ChatGPT plugin. This method requires an OpenAPI specification, documenting the API. For more information, see [API based Message extension](build-api-based-message-extension.md).
+
+* **Bot based message extension**:If you want a one-on-one conversational experience in addition to their plugin, they can create a new message extension from a bot using the Teams Toolkit. For more information, see [Bot based Message extension](build-bot-based-message-extension.md).
```

Comment:
Is this the first reference to the term "plugin" in this article? If so, should probably provide more context about Copilot plugins.